src/Controller/HomeController.php

```diff
@@ -21,6 +21,6 @@ class HomeController extends AbstractController
      */
     public function index()
     {
-        return $this->twig->render('Home/index.html.twig');
+        return $this->twig->render('User/welcome.html.twig');
```

Comment:
it's better to put the templates that a controller renders in a folder with the name of the controller (HomeController => renders templates in "Home" folder)

Reply:
ok correction done